cimgui_funcs.go

```diff
@@ -2274,8 +2360,8 @@ func GetStyleColorName(idx Col) string {
 	return C.GoString(C.GetStyleColorName(C.ImGuiCol(idx)))
 }
 
-func GetStyleColorVec4(idx Col) ImVec4 {
-	return newImVec4FromCPtr(C.GetStyleColorVec4(C.ImGuiCol(idx)))
```

Comment:
Stuff like this sure looks ugly... I'd rather see a code like this being generated:

```go
v := &ImVec4{}
v.fromC(*C.GetStyleColorVec4(C.ImGuiCol(idx)))
return v
```

And by the way, we shouldn't return a pointer here, but a value, so this should be...

```go
v := ImVec4{}
v.fromC(*C.GetStyleColorVec4(C.ImGuiCol(idx)))
return v
```

Reply:
right

Reply:
ok, done.
@eliasdaler may I ask you for review of this PR?

Reply:
> And by the way, we shouldn't return a pointer here, but a value, so this should be...

oh, sorry, hadn't seen that.
so, imo, we should return pointer anyway, since Dear ImGui returns a pointer.
My suppose is, that the pointer holds the value that imgui will change when the style color gets changed, doesn't it?

Reply:
Ah, I see. Then yeah, we should return a pointer.